### DISCUSSION TITLE
Is there a way to add a placeholder?

### DISCUSSION THREAD

**Discussion by Roxanoel** [josefarias/hotwire_combobox#229] [General]
I am using the combobox in a form and I would love to be able to set a placeholder in the underlying input field, but I don't want to actually prefill the box with a valid value. Is there a way to do this, or a plan to implement this feature?

**Comment by josefarias** [comment]
Hey @Roxanoel, yes! You should be able to add a placeholder by passing `placeholder: "Your text here"` to the combobox. E.g. `f.combobox :foo, [options], placeholder: "something"`. Let me know if you run into trouble!

**Reply by Roxanoel** [reply]
I'm using an async combobox, and this doesn't work when I pass `placeholder: "Placeholder text"` as an option.
